src/main.cpp

```diff
@@ -125,6 +125,18 @@ void handleInput (unsigned char key, int x, int y) {
       break;
     case 'h':
       displayUsage();
+      break;
+    case '1':
+      cursor->setShape(1);
```

Comment:
the shape values were off by 1 for the keys and the encoded shapes, I changed where that shows up to make it more clear